src/Microsoft.Health.SqlServer/Features/Schema/SchemaUpgradeRunner.cs

```diff
@@ -70,7 +70,7 @@ public async Task ApplyBaseSchemaAsync(CancellationToken cancellationToken)
             _logger.LogInformation("Completed applying base schema");
         }
 
-        private async Task ExecuteSchemaAsync(string script, CancellationToken cancellationToken)
+        public async Task ExecuteSchemaAsync(string script, CancellationToken cancellationToken)
```

Comment:
Perhaps also change the name to `ExecuteScriptAsync`. There doesn't look like the input needs to be schema specific

Reply:
Good point, Thanks! Fixed the method name.